elements/howto-menu/demo.html

```diff
@@ -0,0 +1,43 @@
+<!doctype html>
+
+<style>
+  /* The tooltip is by default unstyled. */
+  howto-menu {
+    display: block;
+    width: 10em;
+  }
+
+  howto-menuitem {
+    display: inline-block;
+    width: 100%;
+    padding: .2em .5em;
+    border: 1px solid lightgrey;
+  }
+
+  /* The menu is closed. */
+  howto-menu[aria-hidden="true"] {
+    visibility: hidden;
+  }
+
+  /* The menu is open. */
+  howto-menu[aria-hidden="false"] {
+    visibility: visible;
+  }
+</style>
+
+<div class="text">
+   <button aria-label="menu" id="menu-btn">open menu</button>
+   <howto-menu aria-labelledby="menu-btn" aria-hidden="true">
```

Comment:
My thinking was that the developer might want to choose whether a menu is visible on the page initially or hidden away. This said, now that I read again the wai-aria guidelines, a visible menu would be rather called a "menubar"... So it would be a different element I guess.